javascript-timers/interval.js

```diff
@@ -0,0 +1,19 @@
+setInterval(countDown, 1.0 * 1000);
+
+var h1 = document.querySelector('h1');
+
+function countDown() {
+  if (h1.textContent === '...') {
+    h1.textContent = '4';
+  } else if (h1.textContent === '4') {
+    h1.textContent = '3';
+  } else if (h1.textContent === '3') {
+    h1.textContent = '2';
+  } else if (h1.textContent === '2') {
+    h1.textContent = '1';
+  } else if (h1.textContent === '1') {
```

Comment:
If you create a variable to decrement over time, I think you could do this without individual code blocks for each number